apps/atomy-q/API/.env.example

```
@@ -104,6 +104,11 @@ AWS_USE_PATH_STYLE_ENDPOINTS=true
 AI_MODE=provider
 AI_PROVIDER=openrouter
 AI_PROVIDER_NAME=openrouter
+AI_LOG_CHANNEL=ai_operations
+# Comma-separated email list, no brackets required. Example:
+# AI_ALERT_RECIPIENT_EMAILS=ops@example.com,owner@example.com
+AI_ALERT_RECIPIENT_EMAILS=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AI_ALERT_RECIPIENT_EMAILS key should go before the AI_LOG_CHANNEL key

(UnorderedKey)